Makefile

```diff
@@ -66,7 +66,7 @@ $(readme-splice) : docs/TESTS.md
 	cp $< $@
 
 # exercises
-exercises/% : code/track.ss code/exercises/%/* code/stub-makefile
+exercises/% : code/track.ss code/exercises/%/* code/stub-makefile code/docs/tests.ss
```

Comment:
Since this content is now spliced in to the exercise readmes. They should be rebuilt when this is changed

Reply:
Perfect